uber/server.py

```diff
@@ -67,32 +60,28 @@ def common_js(self):
 
     static_views = StaticViews()
 
-_sections = [path.split('/')[-1][:-3] for path in glob(os.path.join(MODULE_ROOT, 'site_sections', '*.py'))
+_sections = [path.split('/')[-1][:-3] for path in glob(os.path.join(c.MODULE_ROOT, 'site_sections', '*.py'))
                                       if not path.endswith('__init__.py')]
 for _section in _sections:
-    _module = __import__('uber.site_sections.' + _section, fromlist=['Root'])
-    setattr(Uber, _section, _module.Root())
+    _module = importlib.import_module('uber.site_sections.' + _section)
+    setattr(Root, _section, _module.Root())
 
-Root = UberShutDown if UBER_SHUT_DOWN else Uber
 
 class Redirector:
     @cherrypy.expose
     def index(self):
-        if AT_THE_CON:
-            raise HTTPRedirect(PATH + '/accounts/homepage')
+        if c.AT_THE_CON:
+            raise HTTPRedirect(c.PATH + '/accounts/homepage')
         else:
-            raise HTTPRedirect(PATH)
+            raise HTTPRedirect(c.PATH)
 
-cherrypy.tree.mount(Root(), PATH, APPCONF)
+cherrypy.tree.mount(Root(), c.PATH, c.APPCONF)
+static_overrides(join(c.MODULE_ROOT, 'static'))
 
```

Comment:
By move these if statements into the background tasks themselves, we can theoretically toggle the config values while the server is running and not need to restart it.